src/Sulu/Bundle/TagBundle/Controller/Exception/ConstraintViolationException.php

```diff
@@ -27,20 +27,13 @@ class ConstraintViolationException extends RestException
     public const EXCEPTION_CODE_NON_UNIQUE_NAME = 1101;
 
     /**
-     * The field of the tag which is not unique.
-     *
-     * @var string
-     */
-    protected $field;
-
-    /**
-     * @param string $message The error message
      * @param string $field The field which is not
-     * @param int $code
      */
-    public function __construct($message, $field, $code = 0)
-    {
-        $this->field = $field;
+    public function __construct(
+        string $message,
+        protected $field,
+        int $code = 0,
```

Comment:
Hoisting type for message and code from the base exception. In PHP8 message must be a string and code must be an integer.